spec/core/ClockSpec.js

```diff
@@ -776,6 +776,26 @@ describe('Clock (acceptance)', function() {
         });
     });
 
+    it('aborts auto ticking when uninstalled, even if installed again synchonrously', async () => {
```

Comment:
This passes in Node but times out in all browsers. I think the problem is that [the clock is set up to overwrite the real timing functions in browsers](https://github.com/jasmine/jasmine/blob/c6b3e947e9673d50d7fd570b6e5cc84b9285b4b7/spec/core/ClockSpec.js#L704), so line 791 calls the mock clock's `setTimeout` instead of the real one.

The best way to fix it is probably to pass a fake global object in all environments, and use `setTimeout.bind(window)` to make Firefox happy. If it's necessary to pass the real global object to the clock, then it should be done unconditionally.

Reply:
👍 Done